src/handlers/spell.handler.ts

```diff
@@ -29,6 +29,8 @@ export class SpellHandler extends MessageHandler {
       player.spells[id].cast(unityPlayer);
     });
 
+    this.socket.emit("cast:spell", JSON.stringify(player.spells[id].name));
```

Comment:
We need to control if the player is allowed to cast a spell before really casting it (e.g if the spells have a currentCooldown > 0)